plc-mc-protocol/plc-mc-protocol.html

```diff
@@ -87,25 +175,64 @@
   /** Returns a new row instance. */
   const getNewRow = (rowNumber, rowValues, $rowTemplate) => {
     if (!$rowTemplate) {
-      $rowTemplate = $('.ia-cloud-plc-mc-addresses-row-template .form-row');
+      $rowTemplate = $('.ia-cloud-plc-mc-addresses-row-template .form-block');
```

Comment:
Should avoid to assignment to function parameter '$rowTemplate'.
In this case, set value at caller.